internal/config/config.go

```diff
@@ -42,8 +42,10 @@ type Config struct {
 // It returns Config struct and an InvalidConfigError if there is an invalid configuration.
 func New() (Config, error) {
 	c := Config{errs: InvalidConfigError{}}
-	c.readFromEnv()
-	c.validate()
+
+	// TODO: remove error from readFromEnv and validate functions
```

Comment:
The error is now included in the `Config` struct. We can remove the `error` from `readFromEnv` and `validate` returns.